### ISSUE TITLE
Failed at func parseItem

### ISSUE THREAD

**Issue by Takiyo0** [TimeForANinja/node-ytsr#106]
`failed at func parseItem: type richShelfRenderer is not known`
OS : win32-x64
node.js : v14.15.1
ytpl : 3.2.4

[7mwqy4k1oe-1612607262401.txt](https://github.com/TimeForANinja/node-ytsr/files/5936653/7mwqy4k1oe-1612607262401.txt)
[adfa726omn-1612606682421.txt](https://github.com/TimeForANinja/node-ytsr/files/5936654/adfa726omn-1612606682421.txt)
[dkuebtiyik-1612607400823.txt](https://github.com/TimeForANinja/node-ytsr/files/5936655/dkuebtiyik-1612607400823.txt)
[dq3klwfbx-1612607259303.txt](https://github.com/TimeForANinja/node-ytsr/files/5936656/dq3klwfbx-1612607259303.txt)
[fxq4c2ml6-1612607533217.txt](https://github.com/TimeForANinja/node-ytsr/files/5936657/fxq4c2ml6-1612607533217.txt)
[njtl0ocoa8-1612607394251.txt](https://github.com/TimeForANinja/node-ytsr/files/5936658/njtl0ocoa8-1612607394251.txt)
[pocyv973ep-1612607299415.txt](https://github.com/TimeForANinja/node-ytsr/files/5936659/pocyv973ep-1612607299415.txt)
[pzzu5jmwr-1612607524628.txt](https://github.com/TimeForANinja/node-ytsr/files/5936660/pzzu5jmwr-1612607524628.txt)
[zbwpfvtczh-1612607287066.txt](https://github.com/TimeForANinja/node-ytsr/files/5936661/zbwpfvtczh-1612607287066.txt)
[zixpcn8zpr-1612607879362.txt](https://github.com/TimeForANinja/node-ytsr/files/5936662/zixpcn8zpr-1612607879362.txt)
[2eolz2m4ya-1612607414101.txt](https://github.com/TimeForANinja/node-ytsr/files/5936663/2eolz2m4ya-1612607414101.txt)
[3t3wswd2m8-1612607263405.txt](https://github.com/TimeForANinja/node-ytsr/files/5936664/3t3wswd2m8-1612607263405.txt)
[5wv8701bu3-1612606339094.txt](https://github.com/TimeForANinja/node-ytsr/files/5936665/5wv8701bu3-1612606339094.txt)

and yes sorry i uploaded everything in dumps because i don't know which file i've to upload

**Comment by brennanwilkes** [comment]
I've also had this problem.

```
Error: type richShelfRenderer is not known
    at parseItem (/home/brennan/gcs-radio/node_modules/ytsr/lib/parseItem.js:69:13)
    at catchAndLogFunc (/home/brennan/gcs-radio/node_modules/ytsr/lib/parseItem.js:76:12)
    at module.exports (/home/brennan/gcs-radio/node_modules/ytsr/lib/parseItem.js:99:46)
    at /home/brennan/gcs-radio/node_modules/ytsr/lib/main.js:54:34
    at Array.map (<anonymous>)
    at Object.module.exports [as default] (/home/brennan/gcs-radio/node_modules/ytsr/lib/main.js:54:25)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

/********************************************************************************************************************************************************************************************************
failed at func parseItem: type richShelfRenderer is not known
pls post the the files in /home/brennan/gcs-radio/node_modules/ytsr/dumps to https://github.com/TimeForANinja/node-ytsr/issues
os: linux-x64, node.js: v14.15.0, ytpl: 3.2.4
********************************************************************************************************************************************************************************************************\

```

[f2sry2h63r-1612994555699.txt](https://github.com/TimeForANinja/node-ytsr/files/5961812/f2sry2h63r-1612994555699.txt)

**Comment by AntoineMaille** [comment]
i have this issue too since yesterday

**Comment by TimeForANinja** [comment]
fixed in https://github.com/TimeForANinja/node-ytsr/commit/8c5dc75e14a84d536fbd6d55855c6afec45e6dc4
available on git & npm as v3.3.0
